src/components/model/post/PostForm/index.tsx

```
@@ -27,29 +26,26 @@
 
-  // 画像アップロードのカスタムフック
-  const {
-    fileInputRef,
-    isUploading,
-    handleDrop,
-    handleFileChange,
-    handleImageButtonClick,
-  } = useImageUpload({
-    onImageUploaded: insertMarkdownAtCursor,
-  });
+  const { startSelect, handleDrop, isUploading, handleFileChange } =
```

ESLint

[error] 'handleFileChange' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)